src/components/expanded-state/unique-token/ZoomableWrapper.android.js

```diff
@@ -83,7 +85,8 @@ export const ZoomableWrapper = ({
   height,
 }) => {
   // eslint-disable-next-line react-hooks/rules-of-hooks
-  const animationProgress = givenAnimationProgress || useSharedValue(0);
```

Comment:
what's the difference between global and non-global?

Reply:
lol there is none 🤣 not sure why i did that.
removed the dupe.